src/Core/FSpot.Core.UnitTest/BrowsablePointerTests.cs

```diff
@@ -0,0 +1,171 @@
+//
+//  BrowsablePointerTests.cs
+//
+// Author:
+//   Daniel Köb <daniel.koeb@peony.at>
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED AS IS, WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using NUnit.Framework;
+using System.Linq;
+using Hyena;
+using System.Collections.Generic;
+
+namespace FSpot.Core.UnitTest
+{
+    [TestFixture]
+    public class BrowsablePointerTests
+    {
+        private IPhoto photo1 = new FilePhoto(new SafeUri("/1"));
+        private IPhoto photo2 = new FilePhoto(new SafeUri("/2"));
+        private IPhoto photo3 = new FilePhoto(new SafeUri("/3"));
+
+        private class BrowsableCollectionMock : IBrowsableCollection
```

Comment:
I'm not sure if we'll need this mock anywhere else, but should we move this out of here and maybe into a folder like 'Mocks'? That way when looking at the test file you only see the tests.